### PR TITLE
RavenDB-15964  Time Series Tabs (RavenDB-15964 + RavenDB-15965)

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/database/query/query.ts [hunk 1]
```diff
@@ -556,14 +556,21 @@ class query extends viewModelBase {
             detectTimeSeries: true, 
             timeSeriesActionHandler: (type, documentId, name, value, event) => {
                 if (type === "plot") {
-                    const chart = new timeSeriesPlotDetails([{
-                        documentId,
-                        value,
-                        name
-                    }]);
+                    const graphs = this.timeSeriesGraphs().filter(graph => _.includes(graph.documentIds, documentId));
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/query/query.ts]: I think this is wrong - let's discuss that today.

**Comment by ml054** [comment]
The impl is wrong. We talked about that vi skype. @Danielle9897 will fix that.

File: src/Raven.Studio/typescript/viewmodels/database/query/query.ts [hunk 2]
```diff
@@ -66,6 +66,18 @@ class highlightSection {
 class timeSeriesTableDetails {
     constructor(public documentId: string, public name: string, public value: timeSeriesQueryResultDto) {
     }
+    
+    isEqual(table: timeSeriesTableDetails) {
+        if (table.documentId != this.documentId) {
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/query/query.ts]: !==

File: src/Raven.Studio/typescript/viewmodels/database/query/query.ts [hunk 3]
```diff
@@ -66,6 +66,18 @@ class highlightSection {
...
+            return false;
+        }
+        
+        if (table.name != this.name) {
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/query/query.ts]: !==

Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/database/query/query.ts]: ok done

File: src/Raven.Studio/typescript/viewmodels/database/query/query.ts (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/database/query/query.ts]: ok done

**Review by ml054 — APPROVED** [review]

**Pull request merged**